### PR TITLE
Azure login generator

### PR DESCRIPTION
**Pull request by ilappe**
## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-1268

## Description
Added a command to generate the customized login url to achieve the Auth code 

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@DeanArbel Doc review completed.

**Comment by ilappe** [comment]
@DeanArbel 
review please...

File: Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py [hunk 1]
```diff
@@ -831,3 +831,24 @@ def get_azure_managed_identities_client_id(params: dict) -> Optional[str]:
...
+        and client.scope \
+        and client.client_id \
+        and client.redirect_uri, 'Please make sure you entered the Authorization configuration correctly.'
+
+    login_url = f'https://login.microsoftonline.com/{client.tenant_id}/oauth2/v2.0/authorize?' \
+                f'response_type=code&scope=offline_access%20{client.scope.replace(" ", "%20")}' \
+                f'&client_id={client.client_id}&redirect_uri={client.redirect_uri}'
+
+    result_msg = f"""### Authorization instructions
+1. Click on the [login URL]({login_url}) to sigh in and grant Cortex XSOAR permissions for your Azure Service Management.
```
Comment (DeanArbel) [inline Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py:848]:
```suggestion
1. Click on the [login URL]({login_url}) to sign in and grant Cortex XSOAR permissions for your Azure Service Management.
```

File: Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py [hunk 2]
```diff
@@ -831,3 +831,24 @@ def get_azure_managed_identities_client_id(params: dict) -> Optional[str]:
...
+You are automatically redirected to a link with the following structure:
```
Comment (DeanArbel) [inline Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py:849]:
```suggestion
You will be automatically redirected to a link with the following structure:
```

File: Packs/AzureDataExplorer/ReleaseNotes/1_2_9.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Azure Data Explorer
+- Added the ***azure-data-explorer-generate-login-url*** command to easy achieve the auth code.
```
Comment (DeanArbel) [inline Packs/AzureDataExplorer/ReleaseNotes/1_2_9.md:4]: @ShirleyDenkberg was this reviewed? This doesn't seem right.
I feel it should be more like this, please confirm.
```suggestion
- Added the ***azure-data-explorer-generate-login-url*** command to easily get the auth code.
```

**Review by DeanArbel — APPROVED** [review]
Looks good, see my comments below.
Please confirm with @ShirleyDenkberg that the releasenotes are ok

File: Packs/AzureDataExplorer/Integrations/AzureDataExplorer/README.md [hunk 4]
```diff
@@ -500,4 +500,34 @@ There are no input arguments for this command.
...
+
+There is no context output for this command.
+
+#### Command Example
+```azure-data-explorer-generate-login-url```
+
+#### Human Readable Output
+
+>### Authorization instructions
+>1. Click on the [login URL]() to sigh in and grant Cortex XSOAR permissions for your Azure Service Management.
```
Comment (DeanArbel) [inline Packs/AzureDataExplorer/Integrations/AzureDataExplorer/README.md:526]: Please correct this in all other docs.
```suggestion
>1. Click on the [login URL]() to sign in and grant Cortex XSOAR permissions for your Azure Service Management.
```

**Pull request merged**